locales/ja/puppetlabs-yumrepo_core.po

```diff
@@ -0,0 +1,34 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2018 Puppet, Inc.
+# This file is distributed under the same license as the puppetlabs-yumrepo_core package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
```

Comment:
Where do these details come from? Should we change them to be something more realistic?